docs/data-management/properties.md

```diff
@@ -6,6 +6,55 @@ sidebar_position: 2
 
 ## Entity properties
 
-### Assignment properties
+## Assignment properties
 
 ## Metric properties
+
+A class of properties that is sourced from Fact SQL.
+
+### Adding via Fact Definition
+
+![Configuring a Fact SQL](/img/properties/metric-property-fact-sql.png)
+
+Add a Metric property to any of your existing Facts or by creating a new one.
+
+This only needs to be configured once and re-use many times for any Metrics created.
+
+### Configure a Metric with a property filter
+
+![Create a new Metric](/img/properties/metric-property-create-metric.png)
+
+Any number of Metrics can be created with one or more Fact properties applied.
+
+![Update a new Metric](/img/properties/metric-property-update-metric.png)
+
+Existing Metrics can also be updated with any configured Metric properties on their Definition.
+
+### Viewing Experiment results
+
+![Update a new Metric](/img/properties/metric-property-experiment-results.png)
+
+Metrics with filters appear normally on Experiment results.
+
+### Exploring Experiment results
+
+When exploring a Metric on an Experiment 
+
+Metric property filters are already applied to the Metric values; as such only Assignment and Entity
```

Comment:
I'd be extra clear here:
"For Metrics that use a Metric Property filter, the filter is already applied and cannot be used to explore. Only Assignment ..."

And then you can say:
"For Metrics that do not use a Metric Property filter, you can explore the results and split by Metric Property values."